projects/storefrontapp-cds-integration/tsconfig.server.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "./tsconfig.app.json",
+  "compilerOptions": {
+    "outDir": "../../out-tsc/app-server",
+    "baseUrl": ".",
+    "module": "commonjs",
+    "types": [],
+    "paths": {
+      "@spartacus/core": ["../../dist/core"],
+      "@spartacus/storefront": ["../../dist/storefrontlib"],
+      "@spartacus/assets": ["../../dist/assets"],
+      "@spartacus/cds": ["dist/cds"]
```

Comment:
Is this the right path? shouldn't it be `../../dist/cds` like the other paths in this file?